docs/python-examples/space-analysis-example.py

```
@@ -0,0 +1,225 @@
+# Space Analysis Example for Python Node
+# This script demonstrates how to perform space analysis using IfcOpenShell
+# in the Python node. Copy this code into a Python node to use it.
+
+import ifcopenshell
+import ifcopenshell.util.element
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)